components/menu/metadata/menu.yml

```diff
@@ -1338,3 +1338,48 @@ examples:
           </ul>
         </div>
       </div>
+  - id: tray-submenus
+    name: Tray submenus
+    description: "When a menu is displayed within a tray, a submenu will replace the tray content when the parent menu item is selected. A submenu displays a back button (labeled by the title of the parent item) at the top of the tray to return the user to the previous level of the menu.
+      The back arrow size scale used with the various menu sizes are small: `200`, medium: `300`, large: `400`, and extra large: `500`."
+    markup: |
+      <div class="spectrum-Tray-wrapper spectrum-CSSExample-dialog" style="background: rgba(0,0,0,0.4);">
+        <div class="spectrum-Modal spectrum-Tray is-open spectrum-Tray">
+          <ul class="spectrum-Menu is-selectableMultiple" role="menu" style="--mod-menu-inline-size: 100%;">
+            <li role="presentation">
```

Comment:
Tell me about this, why do we use role of presentation here?

Reply:
This was the existing markup used for section headers and I think I understand now why it was done. It's because the component is using the `menu` role, and the `li` no longer belongs to that role. The `<li>` is just used to house the heading w/ its list of menu items and the dividers. And it doesn't really function as a list item (_but needs to be one of course because other elements can't be children of a_ `ul`).